src/plugins/memdump/userhook.cpp

```diff
@@ -186,9 +187,10 @@ void memdump::userhook_init(drakvuf_t drakvuf, const memdump_config* c, output_f
     {
         drakvuf_load_dll_hook_config(drakvuf, c->dll_hooks_list, c->print_no_addr, &this->wanted_hooks);
     }
-    catch (int e)
+    catch (UserHookException& exc)
     {
-        fprintf(stderr, "Malformed DLL hook configuration for MEMDUMP plugin\n");
+        std::cerr << "Loading DLL hook configuration for MEMDUMP plugin failed" << std::endl
```

Comment:
```suggestion
        std::cerr << "Loading DLL hook configuration for MEMDUMP plugin failed\n"
```
`std::endl` isn't a simple replacement for `\n`, it also performs a flush and we don't need it here :)

Reply:
eh, I guess. although it's still a cerr and unbuffered

Reply:
That's why I asked to remove `std::endl` in the middle, not at the end :)